examples/ibc/interchain_accounts_test.go

```diff
@@ -268,12 +268,12 @@ func TestInterchainAccounts(t *testing.T) {
 	chain1Chans, err := r.GetChannels(ctx, eRep, chain1.Config().ChainID)
 	require.NoError(t, err)
 	require.Equal(t, 1, len(chain1Chans))
-	require.Equal(t, "STATE_CLOSED", chain1Chans[0].State)
+	require.Subset(t, []string{"STATE_CLOSED", "Closed"}, []string{chain1Chans[0].State})
 
 	chain2Chans, err := r.GetChannels(ctx, eRep, chain2.Config().ChainID)
 	require.NoError(t, err)
 	require.Equal(t, 1, len(chain2Chans))
-	require.Equal(t, "STATE_CLOSED", chain2Chans[0].State)
+	require.Subset(t, []string{"STATE_CLOSED", "Closed"}, []string{chain2Chans[0].State})
```

Comment:
I noticed the go relayer and the hermes relayer return different states, this just accounts for either of them